dockerfiles/zowe-release/amd64/zowe-v1-lts/Dockerfile

```diff
@@ -118,19 +118,19 @@ EXPOSE 8544/tcp
 ENV NODE_HOME='/usr/local'
 ENV ZOWE_JAVA_HOME='/usr'
 
-ENV ZOWE_ZOSMF_HOST='zosmf.host.com'
-ENV ZOWE_ZOSMF_PORT='11443'
-ENV ZOWE_ZSS_HOST='zss.host.com'
-ENV ZOWE_ZSS_SERVER_PORT='00000'
+#ENV ZOWE_ZOSMF_HOST='zosmf.host.com'
+#ENV ZOWE_ZSS_HOST='zss.host.com'
+#ENV ZWED_agent_host='zwed.host.com'
+#ENV ZOWE_EXPLORER_HOST='explorer.host.com'
+ENV ZOWE_IP_ADDRESS=0.0.0.0
 
-ENV ZWED_agent_host='zwed.host.com'
-ENV ZWED_agent_http_port='00000'
+ENV ZOWE_ZOSMF_PORT='11443'
+#ENV ZOWE_ZSS_SERVER_PORT='00000'
+#ENV ZWED_agent_http_port='00000'
 
-ENV ZOWE_EXPLORER_HOST='explorer.host.com'
-ENV ZOWE_IP_ADDRESS=0.0.0.0
+#ENV PKCS11_TOKEN_NAME='ZOWE.ZSS.JWTKEYS'
```

Comment:
For a minimal install with SSO these variables are required, otherwise zowe-setup-certificates will not generates the APIML/ZOSMF JWT cert